components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -99,23 +99,41 @@ extension Nimbus: NimbusEventStore {
 
 extension Nimbus: FeaturesInterface {
     public func recordExposureEvent(featureId: String) {
-        // First we need a list of the active experiments that are enrolled.
-        let activeExperiments = getActiveExperiments()
-
-        // Next, we search for any experiment that has a matching featureId. This depends on the
-        // fact that we can only be enrolled in a single experiment per feature, so there should
-        // only ever be zero or one experiments for a given featureId.
-        if let experiment = activeExperiments.first(where: { $0.featureIds.contains(featureId) }) {
+        // First, we get the enrolled feature, if there is one, for this id.
+        if let enrollment = getEnrollmentByFeature(featureId: featureId),
+           // If branch is nil, this is a rollout, and we're not interested in recording
+           // exposure for rollouts.
+           let branch = enrollment.branch
+        {
```

Comment:
Rewritten `recordExposureEvent` in terms of simpler and cheaper `getEnrollmentByFeature`. Tests still pass.